base/libc.jl

```diff
@@ -159,7 +159,7 @@ function strftime(fmt::AbstractString, tm::TmStruct)
     if n == 0
         return ""
     end
-    return String(pointer(timestr), n)
+    return unsafe_string(pointer(timestr), n)
```

Comment:
Probably.  But there are also a lot of `unsafe_string(pointer(buffer))` usages that are not so easily changed, because they use nul-termination to compute the string length.